### DISCUSSION TITLE
Using emmet_language_server

### DISCUSSION THREAD

**Discussion by skoch13** [LazyVim/LazyVim#1296] [Q&A] [answered]
Has anyone succeeded installing and using `emmet_language_server`? `emmet_ls` worked without any additional configuration, while `emmet_language_server` even being attached and started produces no suggestions.

**Comment by skoch13** [comment, marked as answer]
Caused by this setting
```
        {
          name = "nvim_lsp",
          entry_filter = function(entry, ctx)
            return require("cmp").lsp.CompletionItemKind.Text ~= entry:get_kind()
          end,
        },
```

**Reply by RHOOPH** [reply]
I'm also trying to configure emmet, can you please share your config, if you got it working. Thanks

**Reply by skoch13** [reply]
Sure, simply install `emmet_language_server` via mason and you're good to go.
